FirebaseAI/Sources/PartsRepresentable+Image.swift

```diff
@@ -63,13 +63,22 @@ enum ImageConversionError: Error {
       return [InlineDataPart(data: data, mimeType: "image/jpeg")]
     }
   }
-#endif
+
+  #if swift(>=6.2)
+    @available(iOS 15.0, macOS 12.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
+    extension NSImage: PartsRepresentable {}
```

Comment:
This produces the warning `Conformance of 'NSImage' to 'Sendable' is only available in macOS 14.0 or newer` on Xcode 26.1 (and potentially other versions of 26).